lib/OpenQA/Git.pm

```diff
@@ -24,12 +24,14 @@ has 'user';
 
 sub enabled {
     my ($self) = @_;
-    return ($self->app->config->{global}->{scm} || '') eq 'git';
+    return undef unless my $app = $self->app;
```

Comment:
Should these `die`, too? Otherwise this just results in a silent failure.

Similar concern for `config`, except it would cause attribute errors.

Reply:
fixed